src/modules/accommodations/entities/accommodations.entity.ts

```diff
@@ -67,7 +67,7 @@ export class Accommodation {
   @OneToMany(() => AccommodationImage, (image) => image.accommodation)
   images: AccommodationImage[];
 
-  @OneToOne(() => AccommodationAmenity)
+  @OneToOne(() => AccommodationAmenity, (amenity) => amenity.accommodation)
```

Comment:
Why was this change required?

Reply:
it appeared when I pulled updates to main and merged into branch. There was a conflict and I left this. Should I remove it?

Reply:
Just checked in the docs: https://orkhan.gitbook.io/typeorm/docs/one-to-one-relations

Seems to okay